qiita_pet/handlers/processing_handlers.py

```diff
@@ -0,0 +1,27 @@
+from tornado.web import authenticated
+
+from .base_handlers import BaseHandler
+from qiita_ware.dispatchable import processor
+from qiita_db.parameters import ProcessedSortmernaParams
+from qiita_ware.context import submit
+
+
+class ProcessHandler(BaseHandler):
+    @authenticated
+    def post(self):
+        study_id = int(self.get_argument('study_id'))
+        preprocessed_data_id = int(self.get_argument('preprocessed_data_id'))
+
+        # We currently do not support defining the parameters
```

Comment:
No, the solution is not ready. I prefer to add it in another PR.

2015-01-01 10:48 GMT-07:00 Antonio Gonzalez <notifications@github.com>:

> In qiita_pet/handlers/processing_handlers.py
> <https://github.com/biocore/qiita/pull/732#discussion-diff-22400795>:
>
> > @@ -0,0 +1,27 @@
> > +from tornado.web import authenticated
> > +
> > +from .base_handlers import BaseHandler
> > +from qiita_ware.dispatchable import processor
> > +from qiita_db.parameters import ProcessedSortmernaParams
> > +from qiita_ware.context import submit
> > +
> > +
> > +class ProcessHandler(BaseHandler):
> > +    @authenticated
> > +    def post(self):
> > +        study_id = int(self.get_argument('study_id'))
> > +        preprocessed_data_id = int(self.get_argument('preprocessed_data_id'))
> > +
> > +        # We currently do not support defining the parameters
>
> @josenavas <https://github.com/josenavas> is the solution ready? Can you
> add?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/732/files#r22400795>.
>



-- 
Jose Navas